python/dgl/distributed/optim/pytorch/sparse_optim.py

```diff
@@ -65,10 +254,20 @@ def step(self):
                 # Note: we cannot skip the gradient exchange and update steps as other
                 # working processes may send gradient update requests corresponding
                 # to certain embedding to this process.
-                idics = th.cat(idics, dim=0) if len(idics) != 0 else \
-                    th.zeros((0,), dtype=th.long, device=th.device('cpu'))
-                grads = th.cat(grads, dim=0) if len(grads) != 0 else \
-                    th.zeros((0, emb.embedding_dim), dtype=th.float32, device=th.device('cpu'))
+                idics = (
```

Comment:
What does `idics` mean?  Need a better name.

Reply:
The code change is by format tools and it is old code. My understanding is the idx is like local node index.

Reply:
Fair enough.